### ISSUE TITLE
a11y fixups

### ISSUE THREAD

**Issue by amygroshek** [Hyperobjekt/untd-site#36]
```
warn ESLintError:
/Users/lougro/repos/untd-site/src/pages/research-library.js
  168:15  warning  Visible, non-interactive elements with click handlers must have at least one keyboard listener

jsx-a11y/click-events-have-key-events
  168:15  warning  Static HTML elements with event handlers require a role

jsx-a11y/no-static-element-interactions
  168:45  warning  Elements with ARIA roles must use a valid, non-abstract ARIA role
                                                                                                                                                                            jsx-a11y/aria-role
  175:23  warning  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to
resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
  175:23  warning  Visible, non-interactive elements with click handlers must have at least one keyboard listener

jsx-a11y/click-events-have-key-events
  175:23  warning  Static HTML elements with event handlers require a role

jsx-a11y/no-static-element-interactions
  175:26  warning  Elements with ARIA roles must use a valid, non-abstract ARIA role
```
